R/correctionsataglance-data.R

```diff
@@ -80,13 +80,13 @@ formatDateRange <- function(startD, endD){
   endD <- formatDates(endD)
   firstOfMonth <- day(startD) == 1
   firstOfMonth_end <- day(endD) == 1
+  numdays <- as.numeric(difftime(strptime(endD, format="%Y-%m-%d"), strptime(startD,format="%Y-%m-%d"), units="days"))
   
   if(firstOfMonth){
     startSeq <- seq(startD, endD, by="1 month")
   } else {
     fromDate <- as.POSIXct(format(seq(startD, length=2, by="month")[2], "%Y-%m-01"))
-    numdays <- as.numeric(difftime(strptime(endD, format="%Y-%m-%d"), strptime(startD,format="%Y-%m-%d"), units="days"))
-    if (numdays<=27) {
+    if (fromDate >= endD) {
```

Comment:
I think this is logical and works better than the arbitrary is it less than a full month (27).